pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ['setuptools<65.2', 'wheel']
```

Comment:
I should be able to verify that the original issue (https://github.com/enthought/traits/pull/1722) got resolved by simply creating an edm environment and pip installing traits locally?

Reply:
Sorry, I missed this. Yes, that should work.

Reply:
yep yep it worked for me locally on python 3.11.4 edm